### DISCUSSION TITLE
How do I remove occluders?

### DISCUSSION THREAD

**Discussion by AtomykAU** [diesel-modding/BeardLib-Editor#731] [Q&A] [answered]
I'm trying to remove a wall on bank heist so I can take a wide screenshot of a certain area, how do I remove the occluders so objects behind where the wall was will render when I take the screenshot?

**Comment by Oreztov** [comment, marked as answer]
You first type "occluder" in the unit search menu and then press backspace on your keyboard, or click the "Delete" button in the selection menu.

**Reply by AtomykAU** [reply]
Didn't work, any ideas why?

**Reply by Oreztov** [reply]
Did you save and restart the level?

**Reply by AtomykAU** [reply]
That worked, thank you!
